coding/python_testing.rst

```diff
@@ -34,6 +34,10 @@ The important things to note in this example are:
 * Test classes must, ultimately, inherit from :class:`unittest.TestCase` in order to be discovered, and it is :ref:`recommended <py-utils-tests>` that :lclass:`lsst.utils.tests.TestCase` be used as the base class when :lmod:`~lsst.afw` objects are involved.
   The tests themselves must be methods of the test class with names that begin with ``test``.
   All other methods and classes will be ignored by the test system but can be used by tests.
+* Specific test asserts, such as :meth:`~unittest.TestCase.assertGreater`, :meth:`~unittest.TestCase.assertIsNone` or :meth:`~unittest.TestCase.assertIn`, should be used wherever possible.
+  It is always better to use a specific assert because the error message will contain more useful detail and the intent is more obvious to someone reading the code.
+  Only use :meth:`~unittest.TestCase.assertTrue` or :meth:`~unittest.TestCase.assertFalse` if you are checking a complex statement that is unsupported by other asserts.
```

Comment:
Also, I hope that spelling out the full name (with library prefix) will not confuse users. I assume it is necessary in order for links to work, but they are used as `self.assertGreater` and so on. Still...the examples should make it clear.